skbio/io/format/genbank.py

```diff
@@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
         elif header == 'FEATURES':
             # merge the two dictionaries together. Note this is only
             # available in Python-3.5
-
             feature_metadata = merge_dicts(parsed, feature_metadata)
         else:
             metadata[header] = parsed
```

Comment:
`locus_name` is required at first line of genbank file (looking at
`_parse_locus`). Just nicer to have seq.metadata['id'] than leaving it
empty, so when we writing it into fasta format, the header line is not
empty.

On Thu, Feb 25, 2016 at 12:20 PM, Jamie Morton <notifications@github.com>
wrote:

> In skbio/io/format/genbank.py
> <https://github.com/RNAer/scikit-bio/pull/5#discussion_r54157696>:
>
> > @@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
> >          elif header == 'FEATURES':
> >              # merge the two dictionaries together. Note this is only
> >              # available in Python-3.5
> > -
> >              feature_metadata = merge_dicts(parsed, feature_metadata)
> >          else:
> >              metadata[header] = parsed
>
> Can you clarify this? The metadata object is just a dictionary, it
> doesn't correspond to the features/intervals in the interval_metadata
> object.
>
> Also what if locus_name is not present? The id will still be empty ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RNAer/scikit-bio/pull/5/files#r54157696>.
>